.github/linters/.markdown-lint.yml

```diff
@@ -27,9 +27,11 @@ MD024:
   allow_different_nesting: true # Multiple headers with the same content
 MD026:
   punctuation: ".,;:!。，；:"  # List of not allowed
+MD028: false                  # No blank lines inside blockquote
 MD029: false                  # Ordered list item prefix
 MD033: false                  # Allow inline HTML
 MD036: false                  # Emphasis used instead of a heading
+MD041: false                  # First line in file should be a top level heading
```

Comment:
This can be overridden on a file level by using a comment in the affected file. I do not think we should disable this on a global level.

Reply:
Ah, that sounds like a better solution. The problem is a quirk of the specific readme so I agree. :)

Reply:
Fixed in latest commit